data/questionnaires/personal-social-and-emotional-development.yml

```diff
@@ -461,9 +461,9 @@ personal-social-and-emotional-development:
           Complete the statement: Effective communication between practitioners and parents will result in BLANK expectations for children.
         assessment_summary: 
         assessment_fail_summary: |
-          When there is effective communication between practitioners and parents, everyone caring for the child will have a full understanding of their needs, preferences and level of development. This will support better understanding of both the child’s current skills and the plan to promote progress. When everyone has realistic expectations, the child will have the freedom to learn in supportive environments. 
+          When everyone has realistic expectations, the child will have the freedom to learn in supportive environments. When there is effective communication between practitioners and parents, everyone caring for the child will have a full understanding of their needs, preferences and level of development. This will support better understanding of both the child’s current skills and the plan to promote progress.  
```

Comment:
Is it this bit you were thinking about rewording? 

Maybe 'When practitioners, parents and carers all have realistic expectations for children, it creates a supportive learning environment where children feel confident and have the freedom to learn. When there is effective communication between practitioners, parents and carers...."